ding/model/template/vae.py

```diff
@@ -0,0 +1,210 @@
+"""Note the following vae model is borrowed from https://github.com/AntixK/PyTorch-VAE"""
+
+import torch
+from torch.nn import functional as F
+from torch import nn
+from abc import abstractmethod
+from typing import List, Callable, Union, Any, TypeVar, Tuple
+
+# from torch import tensor as Tensor
+Tensor = TypeVar('torch.tensor')
```

Comment:
`torch.Tensor` can be better, and move this definition to `ding/utils/type_helper.py`